src/axom/slic/core/LogStreamStatusMonitor.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ ******************************************************************************
+ *
+ * \file LogStreamStatusMonitor.cpp
+ *
+ * \brief Implementation of the LogStreamStatusMonitor class.
+ *
+ ******************************************************************************
+ */
+
+#include <mpi.h>
```

Comment:
This include also needs to be guarded by `AXOM_USE_MPI`  (caught by failing CI in the non-mpi jobs).